compiler/resolution/resolveFunction.cpp

```diff
@@ -1579,6 +1632,21 @@ static void insertCasts(BaseAST* ast, FnSymbol* fn, Vec<CallExpr*>& casts) {
                 // We've replaced the move with no-init/assign, so remove it.
                 call->remove();
 
+/*
+              This code is apparently unnecessary
```

Comment:
Do we just not test it, or did you find a test that should check it and found that this branch's operations don't impact it?

Reply:
Except you copied it down below, I'm confused

Reply:
I originally had it in both places but that passed testing, so I only needed it below. Please ignore all the commented-out code - it's just breadcrumbs for me in case something big changes in testing.